### ISSUE TITLE
ótima iniciativa

### ISSUE THREAD

**Issue by Rikintosh** [MaxPresi/FalloutNewVegas_BR#2]
Graças a Deus alguem que faz uma tradução com boa vontade, fiquei indignado como os caras da gv tiveram coragem de lançar aquela bosta ao publico. Fora que nos consoles não tem til (~), e a traduçao das dlcs (nos consoles) causa um bug que vc passa a ganhar apenas 1 xp ao invés de xp normal. Eu corrigi o que pude a muitos anos atras, e mandei pros caras da tcxs...

Baixei aqui, instalei o jogo novamente e vou testar, se eu encontrar problemas, vou anotando conforme jogo, em seguida posto tudo o que eu encontrar aqui

**Comment by MaxPresi** [comment]
Eu não tenho como testar nos consoles. Nem sabia que dava pra colocar mod da nexus lá. O arquivo que está aqui, não funciona se não tiver o ESM pra mesclar o dicionário, e eu não vou postar o ESM aqui pois é muito grande, vou colocar no nexus direto quando terminar. Em relação aso gráficos eu acho q já não faz sentido, pois a intenção não é localizar o jogo, e sim apenas traduzir textos presentes no jogo

**Comment by Rikintosh** [comment]
Sim, eu mesclei o arquivo. Não são todos os mods que funcionam, mas alguns. Nos consoles, o menu nunca é traduzido (deve ser outro arquivo), e ã e õ, não existem, mas uma hora eu investigo isso e corrijo. Tenho devkits de ps3 e xbox 360
